packages/core/src/hooks/useContractRead.test-d.ts

```diff
@@ -0,0 +1,66 @@
+import { tokenAddress } from "./../../test/devnet";
+import { test_abi } from "../../test/test-abi";
+import { assertType, describe, it } from "vitest";
+import { UseContractReadProps, useContractRead } from "./useContractRead";
+import { FunctionRet } from "abi-wan-kanabi/dist/kanabi";
+
+type TAbi = typeof test_abi;
+
+const common_props = {
```

Comment:
```suggestion
const commonProps = {
```

Let's use camelCase everywhere